index.html

```diff
@@ -69,6 +69,149 @@ <h1 class="header_job">I’m a software developer</h1>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+          </section>
+        </div>
+      </section>
+
+      <section id="about">
+        <section class="about_top">
+          <h2 class="about_title">About Me</h2>
+          <p class="about_text">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <button class="about_button">Get my Resume</button>
+        </section>
+        <section class="about_bottom">
+          <div class="languages">
```

Comment:
- **_[OPTIONAL]_** You have done a fantastic job implementing the about section 👏🏽. However, you can improve the look of your components by giving some margin from the corners 📦 as in the given design 👇🏽.
### Figma design:
![image](https://user-images.githubusercontent.com/43172164/205278679-e2059d4a-eb9b-47ae-8cfb-88c623fccb79.png)
### Your design:
![image](https://user-images.githubusercontent.com/43172164/205278745-1dcfd750-1438-4e44-bcb2-e5ab8dd06192.png)